modules/manually-creating-alibaba-ram-user.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_alibaba/manually-creating-alibaba-ram.adoc
+
+:_content-type: PROCEDURE
+[id="manually-creating-alibaba-ram-user_{context}"]
+= Creating the required RAM user
+
+// https://github.com/openshift/cloud-credential-operator/pull/412/files#diff-2480a11ca4927139d6eaa9883946b6f4cb38358cd98def8c57dd73e9319dbc9cR232
+
+When you configure the Resource Access Management (RAM) user, be sure to consider the following requirements:
+
+* Attach the `AdministratorAccess` and `AliyunBSSFullAccess` policies to a RAM user to ensure that the account has sufficient permission to create the {product-title} cluster. This policy grants permissions to manage all Alibaba Cloud resources. The user must have the following permissions:
```

Comment:
As commented [here](https://github.com/openshift/openshift-docs/pull/41083/files#r802969761), `AdministratorAccess` is enough to create the cluster.
cc @gauravsingh85